src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,17 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
```

Comment:
According to the checklist, use abstract references instead of specific ones where possible to leverage polymorphism. Instead of declaring 'Truck', 'Bulldozer', and 'Excavator' specifically, use 'Machine' as the reference type for these objects.